contracts/Treasury.sol

```diff
@@ -153,7 +155,7 @@ contract Treasury is ContractGuard, Operator {
             return (false, 'Treasury: there is no seigniorage to be allocated');
         }
 
-        uint256 cashSupply = IERC20(cash).totalSupply();
+        uint256 cashSupply = IERC20(cash).totalSupply().sub(seigniorageSaved);
```

Comment:
this is minor but can you make `seigniorageSaved` `public` so people doing analysis can easily grab what the `cashSupply` is?

Reply:
just found `getReserve` -- nevermind :)